src/ImageSharp/Formats/IImageDecoder.cs

```diff
@@ -17,17 +17,16 @@ public interface IImageDecoder
         /// <typeparam name="TPixel">The pixel format.</typeparam>
         /// <param name="configuration">The configuration for the image.</param>
         /// <param name="stream">The <see cref="Stream"/> containing image data.</param>
-        /// <returns>The decoded image of a given pixel type.</returns>
```

Comment:
Might be worth to keep the warning, see #1064.

Reply:
I think the documented return type is adequate for now.